OSpackages.sh

```diff
@@ -415,6 +424,10 @@ do
                   ;;
             esac
             ;;
+      -X|--cross-distro)
+            ISCROSSDISTRO=TRUE
+            esac
```

Comment:
This `esac` seems like it may be extraneous here?

Reply:
Heh... came back from pushing the fix to finding this. Any way, yes, happened when I was resolving conflicts in the damned GUI. 😛